.github/workflows/auto_label_prs.yaml

```diff
@@ -14,6 +14,14 @@ jobs:
   set_labels:
     runs-on: ubuntu-latest
     steps:
+      - name: Update gh
```

Comment:
"Harden runner" should really be first -- maybe change egress-policy to `audit` temporarily too.